straxen/plugins/peak_shadow.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import strax
+import numba
+export, __all__ = strax.exporter()
+
+
+@export
+@strax.takes_config(
+    strax.Option('pre_s2_area_threshold', default=1000,
+                 help='Only take S2s large than this into account'
+                 ' when calculating PeakShadow [PE]'),
+    strax.Option('time_window_backward', default=int(3e9),
+                 help='Search for S2s causing shadow in this time window [ns]'),
+    strax.Option('skip_drift_time', default=0,
+                 help='Deduct max drift time to avoid peak interference in a single event [ns]'))
```

Comment:
Maybe we should change this in a similar way as in: 

https://github.com/XENONnT/straxen/blob/090e3faa152c9ab5eaf795ea7b7c16fa1b5eccef/straxen/plugins/event_processing.py#L30-L38

and 

https://github.com/XENONnT/straxen/blob/090e3faa152c9ab5eaf795ea7b7c16fa1b5eccef/straxen/plugins/event_processing.py#L70-L74

Then you can add a another option exclude_drift_time True/False

Reply:
Thanks. I will change it.